textproc/lilypond/Portfile

```diff
@@ -17,113 +16,96 @@ long_description    Lilypond is a unix-based automated engraving system that \
-        scripts/auxiliar/coverage.py \
-        scripts/auxiliar/find-superfluous-includes.py \
-        scripts/auxiliar/fixcc.py \
-        scripts/auxiliar/makelsr.py \
-        scripts/auxiliar/musicxml_generate_intervals.py \
-        scripts/auxiliar/musicxml_generate_keys.py \
-        scripts/auxiliar/musicxml_generate_timesignatures.py \
-        scripts/auxiliar/node-menuify.py \
-        scripts/auxiliar/prepare-web-media.py \
-        scripts/auxiliar/readlink.py \
-        scripts/auxiliar/skyline_viewer.py \
-        scripts/auxiliar/strip-whitespace.py \
-        scripts/auxiliar/tely-gettext.py \
-        scripts/auxiliar/texi-langutils.py \
-        scripts/auxiliar/texi-skeleton-update.py \
-        scripts/auxiliar/translations-status.py \
-        scripts/auxiliar/update-snippets.py \
-        scripts/build/pytt.py }
-    
-    foreach file ${files} {
-        reinplace s|/usr/bin/env\\\ python|${configure.python}| ${worksrcpath}/$file
-        reinplace s|/usr/bin/python|${configure.python}| ${worksrcpath}/$file
-    }
-    
-    # Use guile18 header files
-    reinplace s|libguile\.h|libguile18.h|g ${worksrcpath}/configure \
-        ${worksrcpath}/lily/include/lily-guile.hh
+build.type          gnu
+configure.cmd       autoconf -f && ./configure
 
-    # help fontforge date calculation
-    reinplace s|20110222|2011-02-22|g ${worksrcpath}/configure
-}
+configure.args-append   --disable-documentation
+configure.args-append   --with-texgyre-dir=${prefix}/share/texmf-texlive/fonts/opentype/public/tex-gyre
+configure.args-append   --with-urwotf-dir=${prefix}/share/fonts/urw-core35-fonts
 
-configure.args-append  --with-ncsb-dir=${prefix}/share/fonts/urw-fonts \
-    --disable-documentation
+configure.env-append    GUILE=${prefix}/bin/guile18
+configure.env-append    GUILE_CONFIG=${prefix}/bin/guile18-config
+configure.env-append    GUILE_TOOLS=${prefix}/bin/guile18-tools
 
-platform darwin {
-    if {${configure.cxx_stdlib} eq "libc++"} {
-        configure.ldflags-append    "-lc++"
-    }
-}
+configure.env-append    LTDL_LIBRARY_PATH=${prefix}/lib
+build.env-append        LTDL_LIBRARY_PATH=${prefix}/lib
+destroot.env-append     LTDL_LIBRARY_PATH=${prefix}/lib
 
-configure.env       LTDL_LIBRARY_PATH=${prefix}/lib \
-    PYTHON_CONFIG=${configure.python}-config \
-    GUILE=${prefix}/bin/guile18 \
-    GUILE_CONFIG=${prefix}/bin/guile18-config \
-    GUILE_TOOLS=${prefix}/bin/guile18-tools
+post-patch {
+    # Use guile18 header files.
+    reinplace \
+        s|libguile\.h|libguile18.h|g \
+        ${worksrcpath}/aclocal.m4 \
+        ${worksrcpath}/configure.ac \
+        ${worksrcpath}/lily/include/lily-guile.hh
 
-build.env           LTDL_LIBRARY_PATH=${prefix}/lib
+    # Correct mf2pt1 binary location.
+    reinplace \
+        s|"-mem=mf2pt1"|"-mem=../../mf2pt1"|g \
+        ${worksrcpath}/scripts/build/mf2pt1.pl
+
+    # Avoid empty locale. This is necessary because gettext versions >= 0.21
```

Comment:
This `reinplace` call is not necessary since the change is already part of the tarball.

Reply:
Thanks for catching that! I'd seen the discussion come by so recently I didn't think it would have made it into the stable release.